src/internal/connector/onedrive/collection.go

```diff
@@ -295,6 +295,7 @@ func (oc *Collection) populateItems(ctx context.Context) {
 				itemID       = *item.GetId()
 				itemName     = *item.GetName()
 				itemSize     = *item.GetSize()
+				metaFileName = itemID
```

Comment:
Should we move this to [line 310](https://github.com/alcionai/corso/pull/2496/files#diff-cd4ad81a441fe5b1a43835fb95436f2a797ad246312beb00704340a57d810c65R310), just for consistency.